src/test/kotlin/interactor/GetHighestApartmentPriceDifferenceCityInteractorTest.kt

```diff
@@ -0,0 +1,67 @@
+package interactor
+
+import FakeData
+import model.CityEntity
+import org.junit.jupiter.api.Test
+
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.TestInstance
+
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class GetHighestApartmentPriceDifferenceCityInteractorTest {
+
+// create an object from FakeData class
+    private lateinit var fakeData: FakeData
+    private lateinit var GetHighestApartmentPrice: GetHighestApartmentPriceDifferenceCityInteractor
+
+    @BeforeEach
+    fun setUp() {
+        fakeData = FakeData()
+        GetHighestApartmentPrice = GetHighestApartmentPriceDifferenceCityInteractor(fakeData)
+    }
+
+//to test execute function we need to list of city , We take it from FakeData class
+
+    @Test
+    fun isEmpty_theListIsEmpty_returnNull(){
```

Comment:
rename the function like: should_return_something_when_something
remove cities variable. you don't have it for all testing